lib/pavilion/pav_test.py

```diff
@@ -487,14 +487,18 @@ def handle_error(_, path, exc_info):
 
     # A process should produce some output at least once every this many
     # seconds.
-    BUILD_SILENT_TIMEOUT = 30
+    NONE_LIST = ['', 'null', 'None', None]
```

Comment:
I'm putting this here so you can see it first. There are four cases.
 1. A valid user specified numerical timeout  - '3'
 2. An invalid user specified timeout (negative, non-int). - 'bob', '-1'
 3. No timeout. - null, 'None', etc
 4. The default timeout. - '' or not set.
 
First off, you don't check and give a good error message for case 2.
Secondly, just take an explicit `null` (which translates to None in python).

Reply:
Check for invalid values is now in the init.